admin_manual/configuration_files/external_storage/google.rst

```diff
@@ -71,6 +71,7 @@ for example ``https://example.com``, without a trailing slash. You need two
 
   https://example.com/nextcloud/index.php/settings/personal
   https://example.com/nextcloud/index.php/settings/admin
+  https://example.com/nextcloud/settings/admin/externalstorages
```

Comment:
Could you add `/index.php` before `/settings`. Then the URL also works on instances that doesn't have shortened URLs 😉 

Thanks